src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/court/CourtService.kt

```diff
@@ -1,321 +1,30 @@
 package uk.gov.justice.digital.hmpps.whereabouts.services.court
 
-import org.slf4j.Logger
-import org.slf4j.LoggerFactory
 import org.springframework.data.domain.Sort
 import org.springframework.stereotype.Service
-import org.springframework.transaction.annotation.Transactional
-import uk.gov.justice.digital.hmpps.whereabouts.dto.CreateBookingAppointment
-import uk.gov.justice.digital.hmpps.whereabouts.dto.Event
-import uk.gov.justice.digital.hmpps.whereabouts.dto.VideoLinkAppointmentDto
-import uk.gov.justice.digital.hmpps.whereabouts.dto.VideoLinkAppointmentSpecification
-import uk.gov.justice.digital.hmpps.whereabouts.dto.VideoLinkBookingResponse
-import uk.gov.justice.digital.hmpps.whereabouts.dto.VideoLinkBookingSpecification
-import uk.gov.justice.digital.hmpps.whereabouts.dto.VideoLinkBookingUpdateSpecification
-import uk.gov.justice.digital.hmpps.whereabouts.dto.prisonapi.ScheduledAppointmentDto
 import uk.gov.justice.digital.hmpps.whereabouts.model.Court
-import uk.gov.justice.digital.hmpps.whereabouts.model.HearingType
 import uk.gov.justice.digital.hmpps.whereabouts.model.VideoLinkAppointment
-import uk.gov.justice.digital.hmpps.whereabouts.model.VideoLinkBooking
 import uk.gov.justice.digital.hmpps.whereabouts.repository.CourtRepository
-import uk.gov.justice.digital.hmpps.whereabouts.repository.VideoLinkAppointmentRepository
-import uk.gov.justice.digital.hmpps.whereabouts.repository.VideoLinkBookingRepository
-import uk.gov.justice.digital.hmpps.whereabouts.services.PrisonApiService
-import uk.gov.justice.digital.hmpps.whereabouts.services.ValidationException
-import java.time.Clock
-import java.time.LocalDate
-import java.time.LocalDateTime
-import javax.persistence.EntityNotFoundException
 
-const val VIDEO_LINK_APPOINTMENT_TYPE = "VLB"
+const val UNKNOWN_COURT_NAME = "Unknown"
 
 @Service
-class CourtService(
-  private val prisonApiService: PrisonApiService,
-  private val courtRepository: CourtRepository,
-  private val videoLinkAppointmentRepository: VideoLinkAppointmentRepository,
-  private val videoLinkBookingRepository: VideoLinkBookingRepository,
-  private val clock: Clock,
-  private val videoLinkBookingEventListener: VideoLinkBookingEventListener
-) {
-  companion object {
-    val log: Logger = LoggerFactory.getLogger(this::class.java)
+class CourtService(private val courtRepository: CourtRepository) {
+  val courtNames: List<String> by lazy {
+    courts.map { it.name }
   }
 
-  @Transactional(readOnly = true)
-  fun getCourtLocations() = courtRepository.findAll().map { it.name }.toSet()
+  val courts: List<Court> by lazy { courtRepository.findAll(Sort.by("name")) }
```

Comment:
Only fetch court data once. It doesn't change.  Additions and revisions to the underlying table enabled_court is likely to be performed by flyway scripts at start up.  All pods should be updated, the first will lock and update tables, the other pods will see the new data.